### DISCUSSION TITLE
Varying prompt position

### DISCUSSION THREAD

**Discussion by hectorBrown** [kovidgoyal/kitty#6319] [Q&A] [answered]
Hi, not really a big deal if I'm honest but since I've noticed this, it has started to get on my nerves.
I use i3wm and when resizing kitty windows the positioning of my prompt changes relative to the top left corner of the window. Out of interest I tried with alacritty and it doesn't experience this issue.
My naive guess is that kitty is trying to fit a whole number of columns/rows in the available space and is adding internal padding to the top/left/bottom/right to account for non-round window width/height. I would much rather just add this padding to the bottom/right so that my prompt always appears in the same place, but I'm not sure if this is possible? Has anyone dealt with something similar?

Apologies if this is a duplicate, I've had a hard time coming up with search terms to target this issue.

Thanks!

**Comment by kovidgoyal** [comment, marked as answer]
https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.placement_strategy

**Reply by hectorBrown** [reply]
Ah! I'm an idiot, thanks so much.
